### DISCUSSION TITLE
Link 8 showing up on PyPlot

### DISCUSSION THREAD

**Discussion by Muhammadsgit** [petercorke/robotics-toolbox-python#411] [General]
![panda3](https://github.com/petercorke/robotics-toolbox-python/assets/17506063/5dc8e9f8-5b39-4c08-a188-3151c2935bc4)

Is it possible to not show the link from the base to the end-effector? 

qt = rtb.jtraj(robot.qr, q_pickup,50)
robot.plot(qt.q, backend='pyplot', movie='panda3.gif')

ERobot: panda (by Franka Emika), 7 joints (RRRRRRR), 1 gripper, geometry, collision
┌─────┬──────────────┬───────┬─────────────┬────────────────────────────────────────────────┐
│link │     link     │ joint │   parent    │              ETS: parent to link               │
├─────┼──────────────┼───────┼─────────────┼────────────────────────────────────────────────┤
│   0 │ panda_link0  │       │ BASE        │ SE3()                                          │
│   1 │ panda_link1  │     0 │ panda_link0 │ SE3(0, 0, 0.333) ⊕ Rz(q0)                      │
│   2 │ panda_link2  │     1 │ panda_link1 │ SE3(-90°, -0°, 0°) ⊕ Rz(q1)                    │
│   3 │ panda_link3  │     2 │ panda_link2 │ SE3(0, -0.316, 0; 90°, -0°, 0°) ⊕ Rz(q2)       │
│   4 │ panda_link4  │     3 │ panda_link3 │ SE3(0.0825, 0, 0; 90°, -0°, 0°) ⊕ Rz(q3)       │
│   5 │ panda_link5  │     4 │ panda_link4 │ SE3(-0.0825, 0.384, 0; -90°, -0°, 0°) ⊕ Rz(q4) │
│   6 │ panda_link6  │     5 │ panda_link5 │ SE3(90°, -0°, 0°) ⊕ Rz(q5)                     │
│   7 │ panda_link7  │     6 │ panda_link6 │ SE3(0.088, 0, 0; 90°, -0°, 0°) ⊕ Rz(q6)        │
│   8 │ @panda_link8 │       │ panda_link7 │ SE3(0, 0, 0.107)                               │
└─────┴──────────────┴───────┴─────────────┴────────────────────────────────────────────────┘

┌─────┬─────┬────────┬─────┬───────┬─────┬───────┬──────┐
│name │ q0  │ q1     │ q2  │ q3    │ q4  │ q5    │ q6   │
├─────┼─────┼────────┼─────┼───────┼─────┼───────┼──────┤
│  qr │  0° │ -17.2° │  0° │ -126° │  0° │  115° │  45° │
│  qz │  0° │  0°    │  0° │  0°   │  0° │  0°   │  0°  │
└─────┴─────┴────────┴─────┴───────┴─────┴───────┴──────┘

**Comment by PAdDyBook** [comment]
I have the same issue. also ik are different. did you solve the problem?

**Reply by Muhammadsgit** [reply]
Not yet and my ik matrix (sol) is different too than the one in the tutorial.
